codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/protocols/ServerHttpProtocolGenerator.kt

```diff
@@ -451,12 +451,12 @@ private class ServerHttpProtocolImplGenerator(
         bindings: List<HttpBindingDescriptor>,
     ) {
         val structuredDataSerializer = protocol.structuredDataSerializer(operationShape)
-        structuredDataSerializer.serverOutputSerializer(operationShape).also { serializer ->
+        structuredDataSerializer.serverOutputSerializer(operationShape)?.let { serializer ->
             rust(
-                "let payload = #T(output)?;",
+                "let payload = #T(&output)?;",
                 serializer
             )
-        }
+        } ?: rust("let payload = \"\";")
```

Comment:
@david-perez I wonder - would you use an empty string here or some other form of "empty" body (I saw somewhere examples with `T=()` as body type...)

Reply:
I think you are referring to https://github.com/awslabs/smithy-rs/blob/main/rust-runtime/aws-smithy-http-server/src/body.rs#L16

A string is fine for now IMO, we can change it in the future if needed.